src/constants.c.in

```diff
@@ -14,14 +14,18 @@ constant_t* rust_get_constants() {
     static constant_t constants[] = {
         // Testable constants
         {{~ #each test_constants as |c|}}
+
```

Comment:
The changes in here are due to `handlebars` changing their behaviour and not rendering a newline in the output if the `each` and its contents are immediately adjacent. 
So then you'd get `#endif#ifdef SIGFOOBAR` and so on.